### ISSUE TITLE
Design a screen for users to choose the DID-method

### ISSUE THREAD

**Issue by satazor** [ipfs-shipyard/pm-idm#254]
## Description

Design a screen for users to choose the DID-method when creating and importing identities.

## Acceptance Criteria

- [ ] Design a simple screen for users to select a DID-method. Here's the info we have to display for each: `name`, `description`, `homepage` and a `icon`

**Comment by satazor** [comment]
Later on #64, we may improve this screen explaining the user why this is important. Alternatively, we may merge both tasks. Something to discuss.

**Comment by andreforsousa** [comment]
@satazor I believe we had this screen before, it was created with placeholders to other DID methods. The latest decision was hiding it from the design comps since it was only supporting did:ipid (not configurable, read only), but we can get back to this, definitely something to think later on.

**Comment by satazor** [comment]
@andreforsousa you are right, but for #249 user story to land, we need to get back to this screen. We will need to discuss how to make this as easy and intuitive as possible.
